website/src/content/docs/docs/libraries/http/operations.md

```diff
@@ -103,6 +105,165 @@ namespace Pets {
 }
 ```
 
+### `@body` vs `@bodyRoot`
+
+The `@body` decorator defines the property which type will be exactly the body. Annotating a property inside a `@body` with applicable metadata (`@header`, `@path`, `@query` for request and `@header`, `@statusCode` for response) will be ignored and log a warning.
+The `@bodyRoot` decorator defines the property where the implicit body resolution will be applied(By default the root of the operation).
```

Comment:
```suggestion
The `@bodyRoot` decorator similarly applies to a property or parameter.  The type of that decorated property or parameter is similarly used to define the request or response body.  If the body type is not a model, the behavior is identical to `@body`. If the body type is a model, instead of exactly defining *only* the body, the model may also contain properties annotated as applicable http metadata. Such properties will be treated as http metadata, will not be included in the request or response body, and will not result in a warning.
```